src/test/java/uk/gov/pay/webhooks/webhook/resource/WebhookResourceIT.java

```diff
@@ -52,7 +52,7 @@ public void shouldCreateAndRetrieveAWebhook() {
                 .body("live", is(true))
                 .body("callback_url", is("https://example.com"))
                 .body("description", is("description"))
-                .body("status", is("ACTIVE"))
+                .body("status", is("active"))
```

Comment:
Is this desired behaviour? Any consumers of this will need to be updated (I know there happens to be one in the admin tool, it could easily lower whatever it gets from the backend so its not case sensitive)

Reply:
If possible I think I'd recommend not changing the external facing statuses to keep them caps -- I'm sure there are inconsistencies with how other enums are presented on our internal endpoints so don't know if there's any exact truth to point to.

Reply:
@sfount wasn't sure about this either 🤷‍♂️ I did this to match what we do with the event_types e.g. https://github.com/alphagov/pay-webhooks/blob/610fd677c5dd8f7925c9258741e9d4d5b9449dd6/src/main/java/uk/gov/pay/webhooks/eventtype/EventTypeName.java#L9 that said if people prefer all caps for this, that should be simpler and could remove some code.

Reply:
@sfount OK I restored the previous behaviour in f1fd2f8 (done as separate commit so easier to change mind if needed)